### PR TITLE
fix(copy): update copy on eligibility start

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes part of #522 

## `eligibility:start`

* [x] Page title to: `Get Started`
* [x] Main heading: `There are three steps to link your transit discount to your bank card.`
* [x] Item 1
  * [x] Label: `1. Sign in to Login.gov`
  * [x] Body: `Login.gov is a safe way to sign in to government services.`
* [x] Item 2:
  * [x] Body: `You will need to verify your age with a driver’s license or ID card.`

image
image

## `eligibility:confirm`

* [x] Page title to: Confirm Eligibility
* [x] Main heading: Let’s see if we can confirm your age with the DMV.
* [x] [Body paragraph](https://docs.google.com/document/d/1xSznvvHuhOdtXUuOq1QNMseyfry3PD5oVSvokI9a-E4/edit#bookmark=id.94514dyrbb8g)
* I did not change the driver license 

image

### CODE REVIEW

File: benefits/locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -469,18 +469,17 @@ msgid "eligibility.pages.index.mst.description"
...
 msgid "eligibility.pages.confirm.dmv.content_title"
-msgstr "Let’s see if we can confirm your age with the DMV"
+msgstr "Let’s see if we can confirm your age with the DMV."
 
 msgid "eligibility.pages.start.dmv.items[0].title"
 msgstr "Provide your California ID number"
 
 msgid "eligibility.pages.start.dmv.items[0].text"
-msgstr "You will verify your age with a driver’s license or ID card"
+msgstr "You will need to verify your age with a driver’s license or ID card"
```
Comment (angela-tran) [inline benefits/locale/en/LC_MESSAGES/django.po:478]:
```suggestion
msgstr "You will need to verify your age with a driver’s license or ID card."
```

**Review by angela-tran — CHANGES_REQUESTED** [review]

**Review by angela-tran — APPROVED** [review]
💯

**Review by thekaveman — CHANGES_REQUESTED** [review]
Please make the same changes to the Spanish PO file with `TODO: `

**Review by thekaveman — APPROVED** [review]
👍

**Pull request merged**